src/main/java/org/sagebionetworks/web/client/widget/discussion/ThreadWidgetViewImpl.java

```diff
@@ -41,4 +55,35 @@ public void addReply(Widget w) {
 	public void clear() {
 		replyListContainer.clear();
 	}
+
+	@Override
+	public void setTitle(String title) {
+		threadTitle.setHTML(title);
```

Comment:
Either use SafeHTMLUtils to make sure that this string is safe, or use Span.setText with the values.